Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -904,9 +919,12 @@ public override void Copy(MSTSWagon copy)
             IsFixGeared = locoCopy.IsFixGeared;
             IsSelectGeared = locoCopy.IsSelectGeared;
             HasWaterScoop = locoCopy.HasWaterScoop;
-            ScoopMinPickupSpeedMpS = locoCopy.ScoopMinPickupSpeedMpS;
-            ScoopMaxPickupSpeedMpS = locoCopy.ScoopMaxPickupSpeedMpS;
-            ScoopResistanceN = locoCopy.ScoopResistanceN;
+//            ScoopMinPickupSpeedMpS = locoCopy.ScoopMinPickupSpeedMpS;
+//            ScoopMaxPickupSpeedMpS = locoCopy.ScoopMaxPickupSpeedMpS;
+//            ScoopResistanceN = locoCopy.ScoopResistanceN;
```

Comment:
There's a lot of commented-out code in this PR. It most cases, it should be removed, as we can always find it again with Git, and there's nothing useful to gain from the code as comments.

Reply:
This is one of a number of changes that will be made to implement additional functionality with filling from water troughs.

Previously my preference has been to leave some code that I am not 100% certain about until it has been through a test cycle in the community. Typically when a feature is completed, all commented code will be removed.

Reply:
We should be able to the AppVeyor automatic builds to test features while they are in PR ("Show all checks" at the bottom of the PR, then "Details", then go to "Artifacts"). Please try this - you should be able to link directly to the zip from a forum post - and let me know how it goes. This is my preferred approach going forward.